Makefile

```diff
@@ -331,14 +331,20 @@ importer-image: PLATFORMS=linux/amd64
 importer-image: PUSH=--load
 importer-image: importer-image-build
 
+# Build multiarch kueue-viz image locally
+.PHONY: local-kueue-viz-image-build
+local-kueue-viz-image-build:
+	BUILDER=$(shell $(DOCKER_BUILDX_CMD) create --use)
+	$(MAKE) kueue-viz-image-build PUSH=$(PUSH)
+	$(DOCKER_BUILDX_CMD) rm $$BUILDER
```

Comment:
Can we apply this changes on `kueue-viz-image` target?

Reply:
Yeah, I was just following the semantics done [here](https://github.com/kubernetes-sigs/kueue/blob/62e94f7434bb63e438373a5f548e67080ccd90fa/Makefile#L190), but adding it directly to the `kueue-viz-image` target means that we wouldn't have to update any CI I would guess